pkg-manager/modules-yaml/src/index.ts

```diff
@@ -95,9 +95,13 @@ const YAML_OPTS = {
   sortKeys: true,
 }
 
+export interface StrictModules extends Modules {
+  registries: Registries
+}
```

Comment:
The difference between `interface extends` and `&`-join of 2 object types is that the former would error if you extend with incompatible fields.